.gitignore

```diff
@@ -31,15 +33,20 @@
 /configh.dos
 /configure
 /debian/remake.debhelper.log
+/dist
+/docs
+/lib
```

Comment:
What is the idea of putting these directories into `.gitignore`? There are plenty of needed files in there, no?

Reply:
In remake-4-2 these directories don't exist, so they are ignored. 

However if you start out in remake-4-3 then switch to remake-4-2, even though the files in the directories are gone, the directories themselves remain. 

I guess this is just the weird way git works.

Reply:
I see, I tend to forget that 4.2 doesn't have the source files in `src`, you're right of course.